cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/event/EventBus.java

```diff
@@ -74,12 +76,29 @@ public void post(Object event) {
      * {@link com.cloudant.sync.event.Subscribe}.
      * </p>
      *
+     * <p>
```

Comment:
Small nitpick but the tags aren't indented properly which makes this hard to follow